packages/main/src/Input.js

```diff
@@ -611,6 +611,7 @@ class Input extends UI5Element {
 		}
 
 		if (isEnter(event)) {
+			this._changeFired = (this.previousValue === this.value);
```

Comment:
One last not working scenario. Type for example "a" and press Enter in order to submit the value (fire change) then delete the value with BACKSPACE, in order to have an empty field, type Enter again-> change event is not fired. The issue is not reproducible if after typing "a" change event is fired on focus out, just when using Enter